### PR TITLE
Move PR#162 from barclamp-tempest to crowbar-openstack

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by jsuchome** [comment]
+1 (needs https://github.com/crowbar/crowbar-openstack/pull/6 for docker  load on nodes)

File: chef/data_bags/crowbar/migrate/tempest/011_add_docker_image.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def upgrade ta, td, a, d
```
Comment (rsalevsky) [inline chef/data_bags/crowbar/migrate/tempest/011_add_docker_image.rb]: should `def upgrade(ta, td, a, d)`

**Comment by rsalevsky** [comment]
Please the one hound issue with my comment and rebase against master to avoid the other violations.

File: chef/data_bags/crowbar/migrate/tempest/011_add_docker_image.rb (hunk 1 above)
Comment (aplanas) [inline chef/data_bags/crowbar/migrate/tempest/011_add_docker_image.rb]: This is what HoundCI says, but is quite inconsistent with the rest of the migrations.  I am not sure what is worse here: have a Hound complain or having a migration different from the others

Comment (rsalevsky) [inline chef/data_bags/crowbar/migrate/tempest/011_add_docker_image.rb]: All new migrations should have the new format, we started already with this.

**Comment by vuntz** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Pull request merged**